internal/email/smtp.go

```diff
@@ -169,10 +173,8 @@ func (b *EmailBuilder) Send() (err error) {
 		if err != nil {
 			return err
 		}
-
-		encoder := base64.NewEncoder(base64.StdEncoding, w)
-		defer encoder.Close()
-		_, err := encoder.Write(attachment.data)
+		fileData := base64.StdEncoding.EncodeToString(attachment.data)
+		_, err = w.Write([]byte(chunkSplit(fileData, MaxLineLength, "\n")))
```

Comment:
shouldnt it be "\r\n" ? the Spec sasys CRLF

Reply:
yes, let's do it right, despite it already solved my problem, but more strict mail servers will still deny....
I commited to my branch, is that all what's necessary?